kuwala/pipelines/google-poi/resources/categories.json

```diff
@@ -0,0 +1,260 @@
+{
+    "administration": {
+        "category": "administration",
+        "tags": [
+            "association_or_organization",
+            "city_hall",
+            "immigration_and_naturalization_service"
```

Comment:
"immigration_and_naturalization_service" is also mapped to "publicService" where it also fits better imo.